### DISCUSSION TITLE
[Support]: Platform architecture not amd64 for image with tag `0.14.1`

### DISCUSSION THREAD

**Discussion by DrissiReda** [blakeblackshear/frigate#15669] [General Support]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

architecture of the image seems wrong, so I can't run it on Kubernetes, docker works fine though

Running this image in containerd yields this error:

```bash
exec /init: exec format error
```
Other versions work properly. this is only specific to the image: `ghcr.io/blakeblackshear/frigate:0.14.1`

### Steps to reproduce

1. run `ctr image pull ghcr.io/blakeblackshear/frigate:0.14.1`
2. run `ctr run --rm --tty ghcr.io/blakeblackshear/frigate:0.14.1 frigate-container`

### Version

0.14.1

### Frigate config file

```yaml
not needed
```

### docker-compose file or Docker CLI command

```yaml
ctr image pull ghcr.io/blakeblackshear/frigate:0.14.1
ctr run --rm --tty ghcr.io/blakeblackshear/frigate:0.14.1 frigate-container
```

### Relevant Frigate log output

```shell
only this but not from frigate: exec /init: exec format error
```

### Relevant go2rtc log output

```shell
none
```

### Operating system

Debian

### Install method

Docker CLI

### Network connection

Wired

### Camera make and model

not relevant

### Screenshots of the Frigate UI's System metrics pages

not needed

**Comment by NickM-27** [comment]
I don't think this is the issue. If you compare the docker inspect values they are identical.

```
docker manifest inspect ghcr.io/blakeblackshear/frigate:0.14.1   
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.oci.image.index.v1+json",
   "manifests": [
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 1632,
         "digest": "sha256:656301bd28c08f15a778f1fe37b8332e0ae8189c6311e0fcc58b1e7a8f8ab310",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 567,
         "digest": "sha256:3dc30ce4d90e7b26ee7ddb60d81da92e1051fa3d2f9586c9c50c96b2ad97d2ee",
         "platform": {
            "architecture": "unknown",
            "os": "unknown"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 2014,
         "digest": "sha256:028ad6c744e22d78ca95089155e64d40ffc55cfd110f0a18a05fd9ee43fd7a19",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 567,
         "digest": "sha256:14d58aec37f25de4b499f1226594249368b181beb42d5b4636c47c0f2b4a2375",
         "platform": {
            "architecture": "unknown",
            "os": "unknown"
         }
      }
   ]
}
```

```
docker manifest inspect ghcr.io/blakeblackshear/frigate:0.14.0
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.oci.image.index.v1+json",
   "manifests": [
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 1632,
         "digest": "sha256:3a39c23b2e3ce65a2cafe04c51dfaab6b5b6dae328ef4fe66494eddd5998ead1",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 567,
         "digest": "sha256:23c852b6a6e22482e588e8f5442022a34b726ea7c04067abef4a9783a015f74a",
         "platform": {
            "architecture": "unknown",
            "os": "unknown"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 2014,
         "digest": "sha256:15ab24306f49389382f01d0e9b82b7d4a32f93d207202f518564753015216818",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.oci.image.manifest.v1+json",
         "size": 567,
         "digest": "sha256:b3c40af1939a6738233cc7eeeed30a3a7f45be3a526f161cb5373de210aadd2f",
         "platform": {
            "architecture": "unknown",
            "os": "unknown"
         }
      }
   ]
}
```

**Reply by DrissiReda** [reply]
Right but maybe it wasn't tagged properly as `amd64` that's what I'm saying

**Reply by NickM-27** [reply]
We can clearly see in the manifest that it is tagged as amd64. The 0.14.1 (and all other version specific tags) container both an amd64 and arm64 container as it is multi-arch.

**Reply by DrissiReda** [reply]
> We can clearly see in the manifest that it is tagged as amd64. The 0.14.1 (and all other version specific tags) container both an amd64 and arm64 container as it is multi-arch.

then what do you think the issue is? Because for the mean time I still am unable to use 0.14.1.
And I was wondering if there was a bug in the CI pipeline, because I've seen this in another project a while ago, where they built amd64 for every arch, despite pushing the image as multiarch. I doubt this is the issue here, since they have different sha256

**Reply by NickM-27** [reply]
I don't know as I am not familiar with kubernetes and this is the first time we are hearing of an issue like this, 4 months after the 0.14.1 release. If the 0.15 beta images work then I don't believe there would be anything different about 0.14.1.
